Corona-Warn-App/src/test/java/de/rki/coronawarnapp/tracing/ui/details/TracingDetailsItemProviderTest.kt

```diff
@@ -184,7 +195,7 @@ class TracingDetailsItemProviderTest : BaseTest() {
         testCollector.latestValue!!.any { it is AdditionalInfoLowRiskBox.Item } shouldBe false
         testCollector.latestValue!!.any { it is BehaviorIncreasedRiskBox.Item } shouldBe false
         testCollector.latestValue!!.any { it is BehaviorNormalRiskBox.Item } shouldBe true
-        testCollector.latestValue!!.any { it is PeriodLoggedBox.Item } shouldBe true
+        //testCollector.latestValue!!.any { it is PeriodLoggedBox.Item } shouldBe true
```

Comment:
why this comment?

Reply:
It shouldn't be. I've commented it to and forgot to uncomment it.